Makefile

```diff
@@ -11,12 +11,22 @@ clean:
 	rm -fr diff
 
 distclean: clean
-	rm -fr bin out
+	rm -fr bin out tmp/.stamp_cargo_regex tmp/regex
 
 diff/chameneos_redux.diff: out/chameneos_redux.txt ref/chameneos_redux.txt
 	mkdir -p diff
 	sed -r 's/^[0-9]+/42/' $< | diff -u ref/chameneos_redux.txt - > $@
 
+bin/regex_dna: src/regex_dna.rs tmp/.stamp_cargo_regex
+	mkdir -p bin
+	$(RUSTC) $(RUSTC_FLAGS) -L ./tmp/regex/target/release $< -o $@
+
+tmp/.stamp_cargo_regex:
+	mkdir -p tmp
+	git clone git://github.com/rust-lang/regex tmp/regex
```

Comment:
We haven't been very good about tagging `regex`, but I just created one that should match the current `crates.io` and updated the `Makefile` to use it (`0.1.27`).